docs/Course Wiki/PHYS Course Offerings/PHYS211.md

```diff
@@ -0,0 +1,50 @@
+# PHYS211
+
+PHYS211 (University Physics: Mechanics) is a 4-credit hour course that is required for both EEs and CEs as a part of the Foundational Mathematics and Science requirement. It is offered in the fall, spring and summer semesters.
+
+## Content Covered
+
+- Kinematics
+- Relative and Circurlar motion
+- Newton's Laws
+- Forces, Friction
+- Work & Energy
+- Center of Mass
+- Momentum
+- Collisions, Impulse and Reference Frames
+- Ratations, Torque, Angular Momentum
+- Simple Harmonic Motion, Pendula, Waves
+- Fluid Statics, Fluid Dynamics
+
+## Prerequisites
+
+- [MATH231](MATH231.md) (corequisite)
+
+The official prerequisites to PHYS211 is credit or concurrent registration in [MATH231](MATH231.md)(While MATH231 is listed as a corequisite for PHYS211, no heavy calculation will be used in PHYS211).
```

Comment:
"The official prerequisites to PHYS211 is" -> "The official prerequisites to PHYS211 are"
Make "while" lowercase, put a space after MATH231.